### PR TITLE
koordlet report gpu model and driver version

### LINKED ISSUE #665: [proposal] koordlet should report gpu card type like(A100\V100) to node lebel
**What is your proposal**:
image

**Why is this needed**:

**Is there a suggested solution, if so, please add it**:

### PR DESCRIPTION
**Pull request by buptcozy**
### Ⅱ. Does this pull request fix one issue?
fixes #665

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/statesinformer/states_device_linux.go [hunk 1]
```diff
@@ -171,6 +191,68 @@ func (s *statesInformer) initGPU() bool {
 	return true
 }
 
+func (s *statesInformer) getGPUDriverAndModel() (string, string) {
+	if s.fakeMVML {
```
Comment (hormes) [inline pkg/koordlet/statesinformer/states_device_linux.go:195]: Is this code only for testing?

Comment (hormes) [reply pkg/koordlet/statesinformer/states_device_linux.go:195]: Consider adding an interface？

```
type nvmlWrapper interface {
  GetGPUDriver() string
  GetGPUModel() string
}
```

and then we can move the mock code to test file.

Comment (buptcozy) [reply pkg/koordlet/statesinformer/states_device_linux.go]: fixed

**Review by eahydra — APPROVED** [review]
/lgtm

**Pull request merged**